build.sbt

```diff
@@ -256,7 +256,7 @@ lazy val client = libraryCrossProject("client")
     )
   )
   .jsSettings(
-    jsVersionIntroduced("0.23.5"),
+    jsVersionIntroduced("0.23.5")
```

Comment:
Actually all the `versionIntroduced` stuff can be removed since it's just to help MiMa know where to start.